src/script/util/util.ts

```diff
@@ -177,6 +179,35 @@ export const trimFileExtension = (filename: string): string => {
   return '';
 };
 
+export const allowedImageTypes = ['image/bmp', 'image/gif', 'image/jpeg', 'image/jpg', 'image/png', '.jpg-large'];
```

Comment:
Mh, not a big fan of pulling this out of the config and into a util. Config seems to be the correct place for this kind of stuff.